### ISSUE TITLE
Penalty weight

### ISSUE THREAD

**Issue by Simracer2018** [avs-code/PREM-Podium-race-E-manager#50]
Thanks for the great Job! I made no problem. Would it be possible in the tool also to take a point penalty weight in the evaluation. For example, in the scoring system maybe build with, so that with place 1 + ?? kg / place 2 + ?? kg / place 3 + ?? kg / from place 4 to place ?? could always enter minus kg kg so that this then also in the overall standings / table behind a driver / team can be seen.
Greeting Armin

**Comment by avs-code** [comment]
Yes, its possible, but is necessary a php programer, it's something that I have not been able to achieve for more than a year. At first a programmer helped me a little, but since he was a volunteer I could not ask for much from him.
I put a post in racedepartmen and assetto corsa forum, but I did not succeed in finding one.
